### PR TITLE
+ Icons integrated * fix kobalt library detection

### PR DESCRIPTION
**Pull request by dmitry-zhuravlev**
- Icons integrated
- fix kobalt library detection

### CODE REVIEW

File: src/icons/KobaltIcons.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package icons
```
Comment (cbeust) [inline src/icons/KobaltIcons.kt:1]: Can you move this class in a `com.beust.kobalt.intellij` package?

**Comment by cbeust** [comment]
Also, I'm going to check in the `.svg` file so that you can generate additional icon sizes if you need them, but I'll do that after merging this PR.

File: src/icons/KobaltIcons.kt (hunk 1 above)
Comment (dmitry-zhuravlev) [inline src/icons/KobaltIcons.kt:1]: No. Otherwise resolution of icons will not work. Should be in "icons" package.

**Comment by dmitry-zhuravlev** [comment]
Ok. But at the moment 13x13 and 16x16 is enough.

**Comment by cbeust** [comment]
Ah didn't know that, LGTM, thanks!

**Pull request merged**